k8s_scripts/LitmusInstall.sh

```diff
@@ -11,11 +18,29 @@ kubectl get pods -n litmus
 echo "---------Services Running in Litmus Namespace----------"
 kubectl get svc -n litmus
 
-# Patching the frontend-service to use the Default port
-kubectl patch svc litmusportal-frontend-service -n litmus -p '{"spec": {"ports": [{"port": 9091,"targetPort": 8080,"name": "http","nodePort":30221}],"type": "NodePort"}}'
+# Waiting for pods to be ready
+kubectl wait --for=condition=Ready pods --all --namespace litmus --timeout=180s
 
-echo "---------Services Running in Litmus Namespace----------"
-kubectl get svc -n litmus
+# Getting The LoadBalancer IP for accessing Litmus-Portal
+kubectl patch svc litmusportal-frontend-service -p '{"spec": {"type": "LoadBalancer"}}' -n litmus
 
-# Waiting for pods to be ready
-kubectl wait --for=condition=Ready pods --all --namespace litmus --timeout=120s
\ No newline at end of file
+IP="";
+# Waiting for LoadBalancer assignment for svc 
+while [ -z $IP ]; 
+do echo "Waiting for end point..."; 
+IP=$(kubectl get svc litmusportal-frontend-service -n litmus --template="{{range .status.loadBalancer.ingress}}{{.hostname}}{{end}}"); 
+[ -z "$IP" ] && sleep 10; 
+done; 
```

Comment:
This will be an infinite loop when the IP is not coming up. Please add timeout and exit an error if it doesn't come after some reties.